internal/rpchelp/helpdescs_en_US.go

```diff
@@ -411,6 +411,7 @@ var helpDescsEnUS = map[string]string{
 	"sendtoaddress-amount":    "Amount to send to the payment address valued in decred",
 	"sendtoaddress-comment":   "Unused",
 	"sendtoaddress-commentto": "Unused",
+	"sendtoaddress-subtractfeefromamount": "Toggles whether the tx fee is subtracted from the payment rather than the change",
```

Comment:
This is not formatted correctly.  Set your editor to auto run `go fmt` on save.

Reply:
Fixed